docs/userguide/ui-concepts.md

```diff
@@ -1,3 +1,151 @@
 # Workflow UI
 
-_Coming soon!_
+This page provides a high level overview of Workflow UI, the companion that allows [Workflow Core](../core-workflow) to drive Android and iOS apps.
+
+## Separation of Concerns
+
+Workflow maintains a rigid separation between its core runtime and its UI support.
+The [Workflow Core](../core-workflow) modules are strictly Swift and Kotlin, with no dependencies on any UI framework.
+Those dependencies are restricted to the Workflow UI modules, as you would expect, with Swift UI relying on iOS, and Kotlin UI on Android.
```

Comment:
`Swift UI` could be mistaken for `SwiftUI` — Apple's native declarative UI framework (similar to Compose, or Blueprint).

Reply:
Yikes, thanks.

Reply:
```
Dependencies on Android and iOS are restricted to the Workflow UI modules, as you would expect.
```